docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastTreeTweedie.cs

```diff
@@ -93,4 +93,5 @@ private class Prediction
             public float Score { get; set; }
         }
     }
-}
\ No newline at end of file
+}
+
```

Comment:
extra lines

Reply:
It's caused by TT system. There is no such a line in my ttinclude and tt files.

---
In reply to: [275562731](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275562731) [](ancestors = 275562731)